zappa/cli.py

```diff
@@ -2305,6 +2309,9 @@ def load_settings(self, settings_file=None, session=None):
         # Additional tags
         self.tags = self.stage_config.get("tags", {})
 
+        # Architectures
+        self.architecture = [self.stage_config.get("architecture", "x86_64")]
+
```

Comment:
seems redundant, since we already initialized `self.architecture` in this method.